Loader/ElFinderLoader.php

```diff
@@ -94,35 +94,60 @@ public function setConfigurator($configurator)
     /**
      * Encode path into hash.
      *
-     * @param Request
      * @param string $path
-     * 
+     *
      * @throws \Exception
-     * 
-     * @return string
+     *
+     * @return mixed
      **/
-    public function encode(Request $request, $path)
+    public function encode($path)
     {
-        $target = ($request->isMethod('POST')) ? $request->get('target') : $request->query->get('target');
+        if ($this->bridge === null) {
+            if (empty($this->instance)) {
```

Comment:
The instance is setted when called the [`load`](https://github.com/helios-ag/FMElfinderBundle/blob/master/Controller/ElFinderController.php#L191) method on `ElFinderLoader`, right?
So when calling this encode function on a pre execution event will always throw an exception, right?